dashboard/app/controllers/admin_search_controller.rb

```diff
@@ -16,7 +16,7 @@ def find_students
 
     # If requested, filter...
     if params[:usernameFilter].present?
-      users = users.where(User.arel_table[:username].matches("%#{params[:usernameFilter]}%"))
+      users = users.where(User.arel_table[:username].matches("#{params[:usernameFilter]}%"))
```

Comment:
What would the performance look like if we got rid of the wildcards entirely and did `User.arel_table[:username].eq(params[:usernameFilter])`? In your test, the improvement was pretty huge by removing the leading %. If we get another large improvement by going to `.eq` it might be worth switching over, considering the large size of the users table and the fact that the CX team says they normally have the exact username they want to search.